packages/internal/src/node/shutdown.js

```diff
@@ -1,6 +1,8 @@
 import process from 'node:process';
 import anylogger from 'anylogger';
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore TODO remove when anylogger has types
```

Comment:
I have been banging my head on this one. The `anylogger` package actually has [type definitions](https://github.com/Download/anylogger/blob/1.0.11/anylogger.d.ts), which seem correct.

Reply:
I had tried using them and kept getting the same "not callable" error. Fwiw, this is the version we're on: https://github.com/Download/anylogger/blob/0.21.0/anylogger.d.ts

I think it's out of scope. But if you're gonna address it be my guest :)